### PR TITLE
Changed List Command to accept "all" and "dep" 

### PR DESCRIPTION
**Pull request by Jeevz10**

### CODE REVIEW

File: local.properties [hunk 1]
```diff
@@ -0,0 +1,8 @@
+## This file must *NOT* be checked into Version Control Systems,
+# as it contains information specific to your local configuration.
+#
+# Location of the SDK. This is only used by Gradle.
+# For customization when using a Version Control System, please read the
+# header note.
+#Mon Sep 24 14:24:11 SGT 2018
+sdk.dir=C\:\\Program Files (x86)\\Android\\android-sdk
```
Comment (jieliangang) [inline local.properties]: this file should be in .gitignore, as it contains info specific to ur local config

File: src/main/java/seedu/address/logic/parser/CliSyntax.java [hunk 2]
```diff
@@ -15,8 +15,9 @@
     public static final Prefix PREFIX_START_DATE = new Prefix("s/");
     public static final Prefix PREFIX_END_DATE = new Prefix("e/");
     public static final Prefix PREFIX_DESCRIPTION = new Prefix("d/");
-    public static final Prefix PREFIX_DEPARTMENT = new Prefix("dep/");
+    public static final Prefix PREFIX_DEPARTMENT = new Prefix("dep");
```
Comment (jieliangang) [inline src/main/java/seedu/address/logic/parser/CliSyntax.java]: will this affect creating/editing persons. as we use dep/ for those commands

Comment (Jeevz10) [reply src/main/java/seedu/address/logic/parser/CliSyntax.java]: yes it will affect

Comment (Jeevz10) [reply src/main/java/seedu/address/logic/parser/CliSyntax.java]: actually as long the prefix is consistently used throughout the code, then it wont really affect right?

Comment (jieliangang) [reply src/main/java/seedu/address/logic/parser/CliSyntax.java]: yes but we want to use dep/ not dep for addEvent and editEvent commands.

**Pull request closed without merging**